sops/__init__.py

```diff
@@ -202,10 +207,17 @@ def main():
         else:
             print("%s doesn't exist, creating it." % args.file)
 
+    if args.unencrypted_suffix:
+        unencrypted_suffix = args.unencrypted_suffix
+    elif 'sops' in tree and 'unencrypted_suffix' in tree['sops']:
+        unencrypted_suffix = tree['sops']['unencrypted_suffix']
```

Comment:
I might fit better into `initialize_tree()` and modify the `UNENCRYPTED_SUFFIX` there.